ps_fuzz/app_config.py

```diff
@@ -150,10 +194,10 @@ def update_from_args(self, args):
                     setattr(self, key, value)
             except AttributeError:
                 logger.warning(f"Attempt to set an undefined configuration property '{key}'")
-                raise
+                sys.exit(1)
```

Comment:
If not strictly necessary, leave the handling on the caller, and don't exit "out of the blue" in the middle of the flow.

Reply:
If there was a reason to do so no problem, just check.